app/code/Magento/InventoryInStorePickupAdminUi/Controller/Adminhtml/Order/NotifyPickup.php

```diff
@@ -110,11 +109,9 @@ private function initOrder(): OrderInterface
         $id = $this->getRequest()->getParam('order_id');
         try {
             $order = $this->orderRepository->get($id);
-        } catch (LocalizedException $e) {
-            if ($e instanceof NoSuchEntityException || $e instanceof InputException) {
-                $this->messageManager->addErrorMessage(__('This order no longer exists.'));
-                $this->_actionFlag->set('', self::FLAG_NO_DISPATCH, true);
-            }
+        } catch (NoSuchEntityException|InputException $e) {
```

Comment:
Where `InputException` is thrown? `OrderRepository` contract doesn't state there is one

Reply:
it is a question for @novikor )) I have just change code, without logic changes